app/boards/arm/planck/planck_rev6.dts

```diff
@@ -44,6 +44,20 @@
             ;
     };
 
+    left_encoder: left_encoder {
```

Comment:
If the keyboard only supports one encoder, should we just call it "encoder"?

Reply:
I had gone with that originally, but rev7 supports two encoders, one on each side. If we eventually support that, it seems like this would be a better fit if they share config. I can switch it back if you like though.

Reply:
Whenever we do that, we can do that in a new `planck_rev7.dts`, etc. For now, keep this relevant to the rev6, so I'd suggest renaming back to just `encoder`.

Reply:
Aye, aye. Renamed to `encoder`.